src/containers/Header/PreviewFlag.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export default function PreviewFlag({ name }) {
+    if(!name) {
+        return null
+    }
+
+    return (
+        <>
+            <span className="preview-flag">
+                { name }
+            </span>
+        </>
+  )
+}
```

Comment:
@queuing4oranges Still missing the empty new line at the end of the file https://github.com/TeskaLabs/asab-webui/pull/433/files#r1250953628

:)